scripts/lua2dox.lua

```diff
@@ -53,7 +53,7 @@ There is hack that will insert the "missing" close paren.
 The effect is that you will get the function documented, but not with the parameter list you might expect.
 ]]
 
-local TYPES = { 'integer', 'number', 'string', 'table', 'list', 'boolean', 'function' }
+local TYPES = { 'integer', 'number', 'string', 'table', 'list', 'boolean', 'function', 'any' }
```

Comment:
`any` was used quite frequently, so I thought it appropriate to add it to `TYPES`.